### DISCUSSION TITLE
No battery entries found in HWiNFOSettings

### DISCUSSION THREAD

**Discussion by CrMo4130** [raiguard/ModernGadgets#253] [Q&A] [answered]
Just upgraded to 1.8.1, and setup everything for the CPU and GPU widgets.  However, I'm not seeing any lines in HWiNFOSettings in regards to the battery values (I'm on a laptop).  I see them listed as follows in HWiNFOIndexList:

193   Battery: Razer Razer Blade Stealth       Battery Voltage                11.228 V
194   Battery: Razer Razer Blade Stealth       Remaining Capacity             18.538 Wh
195   Battery: Razer Razer Blade Stealth       Charge Level                   50.5 %
196   Battery: Razer Razer Blade Stealth       Charge Rate                    -9.471 W

Any chance we can get these line items entered so that the battery widget is useful again?  Thanks!

EDIT:
Refreshed widget and the values reappeared.  I feel silly...

**Comment by raiguard** [comment, marked as answer]
Battery Meter doesn't use HWiNFO at all, it uses ActiveNet, which is just as if not more accurate.
